testcases/crypto/rsa_func.c

```diff
@@ -373,7 +374,8 @@ CK_RV do_EncryptDecryptImportRSA(struct PUBLISHED_TEST_SUITE_INFO *tsuite)
     char *s;
 
     // begin testsuite
-    testsuite_begin("%s Encrypt Decrypt Import.", tsuite->name);
+    testsuite_begin("%s Encrypt Decrypt Import%s.", tsuite->name,
+                    me_only ? " (ME-formatpo)" : "");
```

Comment:
Not sure about this string "ME-formatpo" ...  has this a special meaning? Or is the "po" some leftover?

Reply:
left over from copy & paste - fixed